### PR TITLE
Add k8s yaml language server for monaco editor

### LINKED ISSUE #110: Add auto complete support for k8s lang to Monako
**resource definition template and auto-complete feature**:

**it will be great**:

**Lens desktop**

### PR DESCRIPTION
**Pull request by pashevskii**
In theory, the commit adds k8s schemas for monaco-editor autocomplete. 
Now, there are some errors related to webpack and electron (but it works fine locally)

image

### CODE REVIEW

File: src/renderer/components/dock/editor-panel.tsx [hunk 1]
```diff
@@ -48,6 +57,24 @@ export class EditorPanel extends React.Component<Props> {
...
+    setDiagnosticsOptions({
+      validate: true,
+      enableSchemaRequest: true,
+      hover: true,
+      completion: true,
+      isKubernetes: true,
+      schemas: [
+        {
+          // Id of the first schema
+          uri: 'https://raw.githubusercontent.com/garethr/kubernetes-json-schema/master/v1.9.9/all.json',
```
Comment (jakolehm) [inline src/renderer/components/dock/editor-panel.tsx:70]: https://github.com/yannh/kubernetes-json-schema/ has more up-to-date schemas. We could pick schema that matches with current k8s cluster.

Or alternatively we could generate schema from k8s api openapi schema on-the-fly. This way it would also contain all CRDs (might make senseto implement in a separate PR).

Comment (pashevskii) [reply src/renderer/components/dock/editor-panel.tsx:70]: I think it is not a critical issue for now, I've used the link just to check how it works. Unfortunately, I can't make it work in our current environment (webpack and electron-builder). 
image 
The error appears on the editor initialization, I've investigated all documentation and a huge number of stackoverflow threads, but it didn't help.

**Comment by Nokel81** [comment]
Closing as this is out of date and has not been updated in a long time. Please feel free to reopen if/when you have time.
